movie/src/components/HumbergerNavbar/HumbergerNavbar.css

```diff
@@ -1,5 +1,7 @@
 .hamburgerMenu {
   position: absolute;
+  width: 20%;
```

Comment:
question: is this not too big?

Reply:
It is not too big. 
but, when I remove it , nothing change. So, I don't need it.